util/promlint/promlint.go

```diff
@@ -265,4 +318,16 @@ var (
 		"peta",
 		"pebi",
 	}
+
+	// Common abbreviations that we'd like to discourage.
+	unitAbbreviations = []string{
```

Comment:
Maybe abbreviations for minutes, hours, days, weeks, milliseconds, microseconds and nanoseconds could/should be added to the list?

Reply:
I think adding `min`, `ms`, `us`, and `ns` would be a good idea. I think hours, days, and weeks are pretty rare.

Reply:
Though minutes, hours, days, weeks are not base units. Perhaps there should be a different check for non-base units? That should also perhaps be addressed in a separate PR since there are lots of non base units.

Reply:
I agree with the non-base unit argument. I've added `ms, us, ns` and omitted `min, hr, day, week`

Reply:
Hours and days aren't that unusual.

Reply:
I agree that they are not unusual, and should be linted. However, the linter does not check for any sort of non-base unit right now. To be complete we should also check for things like fahrenheit, kelvin, rankine, inches, yards, bits, calories, pounds, ounces, etc. Is it ok to do that as part of a separate PR?

Reply:
We're already checking for MB and kB, how is that different from minutes, hours and days?

Reply:
Fair point, I expanded the unit linting to check several non base units.